Sources/App/Features/Slots/Migrations/Slot+Migration+v1.swift

```diff
@@ -0,0 +1,52 @@
+import Fluent
+
+class SlotMigrationV1: AsyncMigration {
+
+    func prepare(on database: Database) async throws {
+        try await database.schema(Schema.slot)
+             .id()
+             .field("start_date", .string, .required)
+             .field("duration", .double)
+             .field("event_id", .uuid, .required, .references(Schema.event, "id"))
+             .create()
+
+        // Insert the slot relation into child entities
+        try await database.schema(Schema.presentation)
+            .field("slot_id", .uuid, .references(Schema.slot, "id", onDelete: .setNull, onUpdate: .cascade))
+            .unique(on: "slot_id")
+            .update()
+
+        try await database.schema(Schema.activity)
+            .field("slot_id", .uuid, .references(Schema.slot, "id", onDelete: .setNull, onUpdate: .cascade))
+            .unique(on: "slot_id")
+            .update()
+
+        // Due to the lack of slots when initially created, presentations need to be "upgraded"
+        // We need to get every presentation, check if it has a slot_id or not, and then create the correct slot
+        // TODO: What i describe above
```

Comment:
So I put this note in here, however...

It probably doesn't actually need to be done that way. Given we're pulling everything to fix the migrations in the PR this is dependent on, it probably makes sense to let this run without some sort of a fancy migration.

If we were to just leave it, the talks would simply need slots creating for them, which would take a couple mins on the admin panel, which we'll end up doing anyway as we want to make slots for coffee etc.